twitter.gt2eb.user.js

```diff
@@ -4,6 +4,7 @@
 // @description   A try to make Twitter look good again
 // @author        schwarzkatz
 // @match         https://twitter.com/*
+// @noframes
```

Comment:
if we exclude all iframes, the tweet analytics are also left out (and they need styling).
```suggestion
// @exclude       https://twitter.com/i/cards/*
```